### PR TITLE
Error on upgrade from older series.

### PR DESCRIPTION
**Pull request by ExternalReality**
When running `juju upgrade-series prepare` a user should NOT be able to upgrade from a newer version to an older version.

### CODE REVIEW

**Comment by ExternalReality** [comment]
!!build!!

**Comment by ExternalReality** [comment]
!!build!!

File: apiserver/facades/client/machinemanager/machinemanager.go [hunk 1]
```diff
@@ -478,3 +479,38 @@ func (mm *MachineManagerAPI) removeUpgradeSeriesLock(arg params.UpdateSeriesArg)
...
+	}
+
+	return nil
+}
+
+// IsSeriesLessThan returns a bool indicating whether the first argument's
+// version is lexicographically less than the second argument's, thus indicating
+// that the series represents an older version of the operating system. The
+// output is only valid for Ubuntu series.
+func isSeriesLessThan(series1, series2 string) (bool, error) {
```
Comment (manadart) [inline apiserver/facades/client/machinemanager/machinemanager.go]: Based on the capitalised name in the comment, I think you may have exported this in _export_test_ after-the-fact.
Don't think we need both `isSeriesLessThan` and `seriesLessThan`.

**Comment by ExternalReality** [comment]
$$MERGE$$

**Pull request merged**